### DISCUSSION TITLE
kitty hangs on typing 'exit <Enter>'

### DISCUSSION THREAD

**Discussion by kdato** [kovidgoyal/kitty#3495] [Q&A]
This might be user error, so I don't want to open an issue just yet.

Here is my terminal output:

darkstar:~$ some_nice_program &
[1] <PID>
darkstar:~$ exit                                                     <-- some_nice_program still runs at this point
exit

Kitty hangs at the second exit. The only way to close kitty is "kitty_mod + w"

Any thoughts? Thanks.

**Comment by kdato** [comment]
Update: I don't know why I get strike-through text here. Wasn't my intention, strike-through does not show while editing the post.

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf.html#opt-kitty.close_on_child_death

**Comment by kdato** [comment]
Thanks Kovid.
